### ISSUE TITLE
CR Request for Web Content Accessibility Guidelines 2.2

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#450]
# Document title, URLs, estimated publication date
title: Web Content Accessibility Guidelines 2.2
editors draft: https://w3c.github.io/wcag/guidelines/22/
tr: https://www.w3.org/TR/WCAG22/
publication date: 6 September 2022

# Abstract
Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines will also often make Web content more usable to users in general.

WCAG 2.2 success criteria are written as testable statements that are not technology-specific. Guidance about satisfying the success criteria in specific technologies, as well as general information about interpreting the success criteria, is provided in separate documents. See Web Content Accessibility Guidelines (WCAG) Overview for an introduction and links to WCAG technical and educational material.

WCAG 2.2 extends Web Content Accessibility Guidelines 2.1 [WCAG21], which was published as a W3C Recommendation June 2018. Content that conforms to WCAG 2.2 also conforms to WCAG 2.0 and WCAG 2.1. The WG intends that for policies requiring conformance to WCAG 2.0 or WCAG 2.1, WCAG 2.2 can provide an alternate means of conformance. The publication of WCAG 2.2 does not deprecate or supersede WCAG 2.0 or WCAG 2.1. While WCAG 2.0 and WCAG 2.1 remain W3C Recommendations, the W3C advises the use of WCAG 2.2 to maximize future applicability of accessibility efforts. The W3C also encourages use of the most current version of WCAG when developing or updating Web accessibility policies.

# Status
This section describes the status of this document at the time of its publication. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/.

This is a Candidate Recommendation of Web Content Accessibility Guidelines 2.2 (WCAG 2.2) from the Accessibility Guidelines Working Group. This version integrates changes in response to comments received on the 21 May 2021 wide review draft and earlier drafts. The Working Group plans to advance past Candidate Recommendation when the Candidate Recommendation Exit Criteria have been met.

To comment, file an issue in the W3C WCAG GitHub repository. Although the proposed Success Criteria in this document reference issues tracking discussion, the Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive). Comments are requested by 30 September 2022. In-progress updates to the document may be viewed in the publicly visible editors' draft.

This document was published by the Accessibility Guidelines Working Group as a Candidate Recommendation Draft using the Recommendation track.

Publication as a Candidate Recommendation does not imply endorsement by W3C and its Members. A Candidate Recommendation Draft integrates changes from the previous Candidate Recommendation that the Working Group intends to include in a subsequent Candidate Recommendation Snapshot.

This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

This document was produced by a group operating under the 1 August 2017 W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.

This document is governed by the 2 November 2021 W3C Process Document. 

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/w3c-wai-gl/2022JulSep/0352.html

# Changes
New features since 2.1: http://w3c.github.io/wcag/guidelines/22/#new-features-in-wcag-2-2
Change log for 2.2: http://w3c.github.io/wcag/guidelines/22/#changelog

# Requirements satisfied
The Working Group used the WCAG 2.1 requirements from https://w3c.github.io/wcag/requirements/22/, which inherit the WCAG 2.0 requirements from https://www.w3.org/TR/wcag2-req/.

The Working Group was careful to structure success criteria to meet requirements inherited from WCAG 2.0, including technology neutral, clear conformance, implementability, applicability to stakeholders, clear benefits (in the Understanding content), backwards and forwards compatible. 

WCAG 2.2 provides additional criteria to address the accessibility of content for use-cases that were a focus of WCAG 2.1. This includes improvements in support for users of small or touch-screen devices, as well as for low-vision users and users with cognitive, language, and learning impairments.

# Dependencies met (or not)
There are no normative references to specifications that are not yet Proposed Recommendations.
Aside from normative references, dependencies on other groups are managed through the wide review process. 

# Wide Review
Wide review was sought on the 21 May 2022 Working Draft, which resulted in a year's worth of comments to process.

Horizontal reviews were sought as comment processing finished: https://github.com/w3c/wcag/issues/2560

# Issues addressed
https://github.com/w3c/wcag/issues?q=is%3Aopen+is%3Aissue+label%3A%22WCAG+2.2%22+-label%3ATechniques+-label%3AUnderstanding+

# Formal Objections
There is one objection to the CfC transition https://lists.w3.org/Archives/Public/w3c-wai-gl/2022JulSep/0318.html but the objector does not consider it a Formal Objection https://lists.w3.org/Archives/Group/group-ag-chairs/2022Sep/0004.html
Another person supported the first objection https://lists.w3.org/Archives/Public/w3c-wai-gl/2022JulSep/0324.html and we have off-list confirmation that it also is not a Formal Objection.

# Implementation
Draft implementation report: https://www.w3.org/WAI/WCAG22/implementation-report/
Exit criteria: https://raw.githack.com/w3c/wcag/2022-09_CR/guidelines/index.html#exit-criteria

# Patent disclosures
https://www.w3.org/groups/wg/ag/ipr

**Comment by plehegar** [comment]
- [Review dashboard](https://www.w3.org/PM/horizontal/review.html?shortname=wcag) (one issue needs to be looked)
- [Normative references](https://labs.w3.org/normative-references/check?url=https%3A%2F%2Fw3c.github.io%2Fwcag%2Fguidelines%2F22%2F) (one link might need an update, an other may need to be moved)
- No TAG review?

**Comment by svgeesus** [comment]
Why is this specification still referencing an obsolete, **1998** working draft version of the sRGB standard, which was published in **1999**?

> [IEC-4WD]
    IEC/4WD 61966-2-1: Colour Measurement and Management in Multimedia Systems and Equipment - Part 2.1: Default Colour Space - sRGB. May 5, 1998. 

should be

> [SRGB]
    [Multimedia systems and equipment - Colour measurement and management - Part 2-1: Colour management - Default RGB colour space - sRGB](https://webstore.iec.ch/publication/6169). URL: https://webstore.iec.ch/publication/6169

**Comment by svgeesus** [comment]
Just checked, [the correct values are indeed being used in the definition of relative luminance](https://w3c.github.io/wcag/guidelines/22/#dfn-relative-luminance) so please update your reference to match.

**Comment by michael-n-cooper** [comment]
@svgeesus PR to update ref: https://github.com/w3c/wcag/pull/2661

**Comment by swickr** [comment]
Transition approved.

**Comment by michael-n-cooper** [comment]
[Publication request](https://lists.w3.org/Archives/Team/webreq/2022Sep/0015.html)

**Comment by svgeesus** [comment]
> @svgeesus PR to update ref: https://github.com/w3c/wcag/pull/2661

Thanks! Merged.

**Comment by michael-n-cooper** [comment]
I missed that there was an item at risk in the WG decision. I have [added that to the spec](https://github.com/w3c/wcag/commit/463f16059053f78c4a3eda400112d594fddcf78d) after the CR exit criteria, and referenced it from the status.

**Comment by swickr** [comment]
Transition approved.

Good that you caught this, Michael.

**Comment by michael-n-cooper** [comment]
[Published](https://www.w3.org/TR/2022/CR-WCAG22-20220906/)
